sources/kube.go

```diff
@@ -36,14 +37,66 @@ var kubeVersions = []string{"v0.3"}
 // Cadvisor port in kubernetes.
 const cadvisorPort = 4194
 
+type PodInstance struct {
+	Pod    string
+	HostIp string
```

Comment:
Pod UID might be a better to differentiate between different instances of the same Pod.